source/dotnet/Test/AdaptiveCards.Test/AllPayloadTests.cs

```diff
@@ -127,6 +131,9 @@ public void TestAllTestCards()
                     "FallbackParsing",
                     "DeepFallback",
                     "EmptyFallbackCard",
+
+                    // These cards have AdpativeCards with styles on them
+                    "ColumnColumnSetContainer.Bleed.json",
```

Comment:
>json [](start = 52, length = 4)

is this supposed to have `.json` on the end?

Reply:
it shouldn't matter as the comparison made later checks if the filename contains the given string, for consistency I'll delete the ".json"

---
In reply to: [270216378](https://github.com/Microsoft/AdaptiveCards/pull/2580#discussion_r270216378) [](ancestors = 270216378)